Simplenote/TagListViewController.swift

```diff
@@ -308,73 +325,56 @@ extension TagListViewController: UITableViewDataSource, UITableViewDelegate {
-        return Section(rawValue: indexPath.section) == .tags
+        return false
     }
 
     func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCell.EditingStyle {
-        let isTagRow = Section(rawValue: indexPath.section) == .tags
-        return isTagRow && tableView.isEditing ? .delete : .none
-    }
-
-    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
-        guard editingStyle == .delete else {
-            return
-        }
-
-        SPTracker.trackTagRowDeleted()
-        removeTag(at: indexPath)
+        return .none
     }
 }
 
 // MARK: - Cell Setup
 //
 private extension TagListViewController {
-    func configure(_ cell: SPTagListViewCell, at indexPath: IndexPath) {
-        guard let section = Section(rawValue: indexPath.section) else {
-            return
-        }
 
-        switch section {
-        case .system:
-            configureSystemCell(cell, at: indexPath)
-        case .tags:
-            configureTagCell(cell, at: indexPath)
-        case .bottom:
-            configureBottomCell(cell, at: indexPath)
-        }
-    }
-
-    func configureSystemCell(_ cell: SPTagListViewCell, at indexPath: IndexPath) {
+    func configureSystemCell(_ cell: Value1TableViewCell, at indexPath: IndexPath) {
         guard let row = SystemRow(rawValue: indexPath.row) else {
             return
         }
 
+        cell.hasClearBackground = true
+        cell.imageTintColor = .simplenoteTintColor
+
         switch row {
         case .allNotes:
-            cell.textField.text = NSLocalizedString("All Notes", comment: "")
-            cell.iconImage = .image(name: .allNotes)
+            cell.title = Localization.allNotes
+            cell.imageView?.image = .image(name: .allNotes)
             cell.accessibilityIdentifier = "all-notes"
         case .trash:
-            cell.textField.text = NSLocalizedString("Trash-noun", comment: "")
-            cell.iconImage = .image(name: .trash)
+            cell.title = Localization.trash
+            cell.imageView?.image = .image(name: .trash)
+            cell.accessibilityIdentifier = nil
         case .settings:
-            cell.textField.text = NSLocalizedString("Settings", comment: "")
-            cell.iconImage = .image(name: .settings)
+            cell.title = Localization.settings
+            cell.imageView?.image = .image(name: .settings)
             cell.accessibilityIdentifier = "settings"
         }
     }
 
-    func configureTagCell(_ cell: SPTagListViewCell, at indexPath: IndexPath) {
+    func configureTagCell(_ cell: TagListViewCell, at indexPath: IndexPath) {
         let tagName = tag(at: indexPath)?.name
 
         cell.textField.text = tagName
-        cell.iconImage = nil
         cell.delegate = self
     }
 
-    func configureBottomCell(_ cell: SPTagListViewCell, at indexPath: IndexPath) {
```

Comment:
I sincerely have no idea **why** I named this `configureBottomCell` instead of `configureUntaggedNotesCell` 😂